pkgs/development/python-modules/ebooklib/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, lxml
+, six
+}:
+buildPythonPackage rec {
+  pname = "ebooklib";
+  version = "0.17.1";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "EbookLib";
+    sha256 = "sha256-/iPiLCgFAZbGjbPnsTsle/OUJtknyzlcbyzBOsETJ/E=";
+  };
+
+  propagatedBuildInputs = [ lxml six ];
+
```

Comment:
Maybe you could explicitly enforce tests by using `checkInputs = [ pytestCheckHook ];` and also use `pythonImportsCheck = [ "ebooklib"];`

Reply:
The package seems to have no tests at all, so the pytest hook wouldn't really help. Added the import check, though.